### PR TITLE
Modal/buy now

### PR DESCRIPTION
**Pull request by chuzhov**
Buy now 1 modal window is added

### CODE REVIEW

File: src/partials/modal/buy-now-1.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                </p>
+            </li>
+            <li class="order__item order--brown-border">
+                <p class="order__name">
+                    milkshake
+                </p>
+            </li>
+        </ul>
+        
+        <form action="" class="order__form">
```
Comment (HollowMax) [inline src/partials/modal/buy-now-1.html:32]: action="" value is empty

File: src/sass/main-page/modal/_buy-now-1.scss [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+}
+
+.order__input-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.order__form {
```
Comment (HollowMax) [inline src/sass/main-page/modal/_buy-now-1.scss:178]: Empty selector

**Review by HollowMax — CHANGES_REQUESTED** [review]
Empty value

File: src/sass/main-page/modal/_buy-now-1.scss (hunk 2 above)
Comment (chuzhov) [reply src/sass/main-page/modal/_buy-now-1.scss]: Deleted

**Pull request merged**